install.sh

```diff
@@ -612,14 +630,19 @@ if [[ "${build_hip_clang}" == true ]]; then
   export PATH=${rocm_path}/bin:${rocm_path}/hip/bin:${rocm_path}/llvm/bin:${PATH}
 fi
 
+# Use RocRoller
+if [[ ! ( "${ID}" == "rhel" && "${VERSION_ID}" == "9.1" ) && ( "${skip_rocroller}" == false ) ]]; then
```

Comment:
You try to skip rocroller for RHEL 9 distro. However, CQE side would always build on all distro since they didn't use install.sh way. Is that ok? Or should you check the distro version in CMakeLists.txt directly?

Reply:
CQE builds should have been moved to use RHEL 9.5, which should work. This is kind of a temporary fix until Math CI also moves to RHEL 9.5.

Reply:
Ok.